### PR TITLE
Fixed some bugs

### PR DESCRIPTION
**Pull request by helloImHai**
- Bug in DeleteCommand caused by changing set to array
- Edit command does not reset the list
- Bug in EditSkillParser

### CODE REVIEW

File: src/main/java/seedu/address/model/ResumeBook.java [hunk 1]
```diff
@@ -225,7 +225,7 @@ public void deleteInternship(Internship key) {
         internships.remove(key);
         for (Item item : resumes) {
             Resume resume = (Resume) item;
-            resume.getInternships().remove(id);
+            resume.getInternships().remove(Integer.valueOf(id));
```
Comment (chrisjwelly) [inline src/main/java/seedu/address/model/ResumeBook.java:228]: Does this do anything special or do you just want to be consistent with the signature of `remove`?

File: src/main/java/seedu/address/logic/parser/EditCommandParser.java [hunk 2]
```diff
@@ -130,7 +130,7 @@ public EditCommand parse(String args) throws ParseException {
                 editSkillDescriptor.setName(ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get()));
             }
             if (argMultimap.getValue(PREFIX_LEVEL).isPresent()) {
-                editSkillDescriptor.setName(ParserUtil.parseName(argMultimap.getValue(PREFIX_LEVEL).get()));
+                editSkillDescriptor.setLevel(ParserUtil.parseLevel(argMultimap.getValue(PREFIX_LEVEL).get()));
```
Comment (chrisjwelly) [inline src/main/java/seedu/address/logic/parser/EditCommandParser.java:133]: Good catch!

**Comment by chrisjwelly** [comment]
> 
> 
>     * Bug in DeleteCommand caused by changing set to array
> 
>     * Edit command does not reset the list

How did you fix these two? I'm not sure if I missed it in the codes changed..

File: src/main/java/seedu/address/model/ResumeBook.java (hunk 1 above)
Comment (helloImHai) [reply src/main/java/seedu/address/model/ResumeBook.java:228]: It's to remove the value id, not the value at index id 💃

File: src/main/java/seedu/address/logic/parser/EditCommandParser.java (hunk 2 above)
Comment (helloImHai) [reply src/main/java/seedu/address/logic/parser/EditCommandParser.java:133]: Thanks m8

**Comment by duongphammmm** [comment]
Nicely done!!

**Pull request merged**